cherrypy/lib/static.py

```diff
@@ -28,6 +29,21 @@ def _setup_mimetypes():
 _setup_mimetypes()
 
 
+def _utf8_content_disposition(disposition, file_name):
+    """Create HTTP header for downloading a file with UTF-8 filename.
+
+    See this and related answers:
+    https://stackoverflow.com/a/8996249/2173868
+    """
+    ascii_name = unicodedata.normalize(
+        'NFKD', file_name).encode('ascii', 'ignore').decode()
```

Comment:
I'm not in favor of using `ignore` for error processing. I think it shadows errors. What is the reason behind using it?

Reply:
Try use it with default "errors" setting (strict), you will get "UnicodeEncodeError: 'ascii' codec can't encode character '\u2603' in position 20: ordinal not in range(128)" when running tests. Thats protect it against 'ascii' codec unknown UTF8 character error.

Reply:
Try use it with default "errors" setting (strict), you will get "UnicodeEncodeError: 'ascii' codec can't encode character '\u2603' in position 20: ordinal not in range(128)" when running tests. Thats protect it against 'ascii' codec unknown UTF8 character error.